RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -100,10 +100,18 @@ private void ReplaceAssignedByValParameterReferences(IModuleRewriter rewriter, s
         private void InsertLocalVariableDeclarationAndAssignment(IModuleRewriter rewriter, string localIdentifier)
         {
             var content = Tokens.Dim + " " + localIdentifier + " " + Tokens.As + " " + _target.AsTypeName + Environment.NewLine
-                + (_target.AsTypeDeclaration is ClassModuleDeclaration ? Tokens.Set + " " : string.Empty)
+                + (IsCtLExprContext(_target) ? Tokens.Set + " " : string.Empty)
                 + localIdentifier + " = " + _target.IdentifierName;
 
             rewriter.InsertBefore(((ParserRuleContext)_target.Context.Parent).Stop.TokenIndex + 1, "\r\n" + content);
         }
+
+        private bool IsCtLExprContext(Declaration target)
+        {
+            var argContext = target.Context as VBAParser.ArgContext;
+            var typeCtxt = argContext.asTypeClause().type().complexType();
+
+            return (typeCtxt is VBAParser.CtLExprContext);
```

Comment:
wouldn't this systematically return `true`? how does it handle an `As` clause for an enum qualified with the project it's defined in? e.g. `VBA.VbMsgBoxResult`?